lib/Client.php

```diff
@@ -203,4 +305,42 @@ private function message($msg)
 
         return $msg;
     }
+
+    private function loadFeatureFlags(): void
+    {
+        $response = $this->httpClient->sendRequest(
```

Comment:
Ok so I tried this locally and it didn't work. I _suspect_ this is a potential problem. We're sending a POST request here, when it should actually be a GET request.